File: Java/Array/dynamic_array.java
import java.util.ArrayList;
public class Main {

    public static void main(String[] args) {
       ArrayList<Integer> nums = new ArrayList<>();
       // add values;
       nums.add(10);
       nums.add(30);
       nums.add(-10);
       // print values 
       for( int i = 0 ; i < nums.size() ; i++ ) {
           System.out.println( i + " idx  value is " + nums.get(i));
       }
       
       // change value 
       nums.set(0, 120);
       
       System.out.println( "idx 0 value now " + nums.get(0));
       
       // remove a particular index value 
       nums.remove(1);
       
       // print all values
       for( int num: nums ) {
           System.out.println(num);
       }
       
    }
}

File: C++/Array/dynamic_array.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
    
       vector < int > nums;
       // add values;
       nums.push_back(10);
       nums.push_back(30);
       nums.push_back(-10);
       // print values 
       for( int i = 0 ; i < nums.size() ; i++ ) {
           cout <<  i  <<  " idx  value is " << nums[i] << endl;
       }
       
       // change value 
       nums[0] =  120;
      
       cout << "idx 0 value now " << nums[0] << endl;
       
       // remove a particular index value 
       nums.erase( nums.begin() + 1 );
       
       // print all values
       for( int num: nums ) {
           cout << num << endl;
       }
       
    return 0;
}
